app/models/allocation.rb

```diff
@@ -16,6 +16,8 @@ class Allocation < ApplicationRecord
 
   enum request_type: { initial: 0, repeat: 1, declined: 2 }
 
+  scope :current_allocations, -> { RecruitmentCycle.find_by(year: Settings.allocation_cycle_year).allocations }
```

Comment:
Might be simpler for this to be `where(recruitment_cycle: RecruitmentCycle.find_by(year: Settings.allocation_cycle_year))`